packages/plugin-ext/src/main/browser/command-registry-main.ts

```diff
@@ -77,7 +77,7 @@ export class CommandRegistryMainImpl implements CommandRegistryMain, Disposable
             throw new Error(`Command with id '${id}' is not registered.`);
         }
         try {
-            return await this.delegate.executeCommand(id, ...args);
+            return this.delegate.executeCommand<T>(id, ...args);
```

Comment:
does try/catch still work after these changes?
there is a comment that it was added for some use case...

Reply:
You're right. I think I removed it reflexively, but `try...catch` is the context where `return await` actually has a function....

Reply:
I've restored the `await` here.